packages/lisk-p2p/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const IP_NUMBER_OF_BLOCKS = 4;
+const IP_MAX_VAL_PER_BLOCK = 256;
+const IP_MIN_VAL_PER_BLOCK = 0;
+const MAX_VAL_PORT = 65535;
+const MIN_VAL_PORT = 1;
+// We disabled no-any because we want to validate the values in response from the network that could be any
+/* tslint:disable: no-any */
+export const validateIp = (ipAddress: any): boolean => {
```

Comment:
maybe it's better to use some schema checking library?

Reply:
@SargeKhan should we use https://www.npmjs.com/package/@types/validator for validator.js?

Reply:
@jondubois what do you think?

Reply:
Are these the only validations which will be required in p2p library or do you see the validators list expanding?

Reply:
I think there will be a few more validations, but there will not be many. But there are some validations which are tricky, like ipv4/ipv6 which is better handled by the library. I just have validation for ipv4 right now and Jon said to have it for ipv6 too

Reply:
For schema check, we use https://github.com/epoberezkin/ajv.
For IP, I think core was using some library, which we can use it too.

Reply:
So, in core, `ip` is used in module and logic and `net` in app.js. which one to use or we should use `validator.js` that covers everything?